NGitLab.Mock/Project.cs

```diff
@@ -38,7 +38,7 @@ public Project(string name)
 
         public string Description { get; set; }
 
-        public string DefaultBranch { get; set; } = Repository.DefaultBranch
+        public string DefaultBranch { get; set; } = GitLabServer.DefaultBranch;
```

Comment:
The logic should be more complex. When the server (Parent) is set for this instance, it should set the default branch value unless there is an explicit value.